suite/src/__tests__/fast/data-feed-api.test.ts

```diff
@@ -15,16 +15,16 @@ import { decode } from 'codeco'
 const ComposeDbUrls = String(process.env.COMPOSEDB_URLS)?.split(',')
 const adminSeeds = String(process.env.COMPOSEDB_ADMIN_DID_SEEDS).split(',')
 
-async function genesisCommit(node: CeramicClient, modelInstanceDocumentMetadata: ModelInstanceDocumentMetadataArgs, anchor: boolean) {
+async function genesisCommit(CeramicNode: CeramicClient, modelInstanceDocumentMetadata: ModelInstanceDocumentMetadataArgs, anchor: boolean) {
```

Comment:
another total nitpick, but convention is for variable names to start with lower case, only types generally start with capital letter.

Reply:
oh yeah you are right, thanks for noticing